### ISSUE TITLE
DualShock3 - Steam Input No Rumble

### ISSUE THREAD

**Issue by KiralyCraft** [ValveSoftware/steam-for-linux#7594]
#### Your system information

* Steam client version (build number or date): Jan 5 2021, 22:30:51
* Distribution (e.g. Ubuntu): Manjaro KDE (Kernel 5.10.2)
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Using a an original DualShock 3 controller connected via bluetooth. Rumble works everywhere, including https://html5gamepad.com/ and native games. I've tested Rise Of the Tomb Raider and it works wonders. I tried playing some titles via Proton 5.13-4 and they either refuse to detect the controller, which is no big deal. Enabling Steam Input to wrap the controller into something the games can use works, but without rumble. Not even the "Identify" button which shows in the general controller settings does anything. It's worth mentioning, the "Identify" button only shows if Playstation controller handling is enabled.

#### Steps for reproducing this issue:

1. Play native game, such as Rise of the Tomb Raider as-is, with a DS3 controller. Works as expected
2. Enable Steam Input handling for the controller
3. Launch the same game via Proton. Controls work, rumble doesn't. Same for GTAIV

**Comment by kisak-valve** [comment]
Hello @KiralyCraft, issues with games run with Proton are being tracked in the [Proton](https://github.com/ValveSoftware/Proton/issues) issue tracker. Let's use this issue report to only focus on the "Identify" button in Steam's settings.

That set aside, Proton feedback is possibly a duplicate of https://github.com/ValveSoftware/Proton/issues/3300.

**Comment by KiralyCraft** [comment]
Oh, quite possible! Sorry for the wrong category, I thought this is a Steam Input issue alone. That other one from Proton resembles mine indeed. 

I've done some more testing however, and ROTR running native with Steam Input enabled exhibits the same issue

**Comment by teconmoon** [comment]
I don't think this adds anything useful to the conversation, but I have this same issue trying to use a DualShock 3 controller with the Steam Deck. For any game, regardless of whether it's running natively or a Windows game running through Proton, Desktop Mode or Game Mode, rumble doesn't work when using a DualShock 3 controller through Steam Input.

**Comment by boobthecat** [comment]
Please fix this

**Comment by teconmoon** [comment]
I was surprised to find recently that rumble worked when I tried to use my DualShock 3 controller. No clue what update might have resolved this as I hadn't tried to use it in ages. Can anyone else confirm?
